tests/integration-tests-api-key.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# This runs the auth integration tests in isolation
+
+set -eux
+set -o pipefail
+
+# Ensure current path is project root
+cd "$(dirname "$0")/../"
+
+QDRANT_HOST='localhost:6333'
+export QDRANT__SERVICE__GRPC_PORT="6334"
+
+./target/debug/qdrant --config-path config/production_auth.yml &
```

Comment:
Is a new `production_auth.yml` file necessary?

Reply:
Not really. I can create it on the fly before starting the service to run the integration tests against it. Do you think that would be better?

Reply:
Let's do that to not mix actual config with tests config. FYI, we can always set the configuration with env variables too https://qdrant.tech/documentation/guides/configuration/#environment-variables

Reply:
Excellent. I've dropped it completely and used env vars instead :+1: .